conda/mo_ants/meta.yaml

```
@@ -0,0 +1,89 @@
+{% set name = "mo_ants" %}
+{% set version = "2.0.0" %}
+{% set revision = "HEAD" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  svn_url: <Insert MOSRS svn url>/ancil/ants/tags/{{ version }}
```

We can allow the URL to be replaced at different sites by using an environment variable like the lfric utils
```suggestion
  svn_url: {{ environ.get('NGMOENVS_MOSRS_MIRROR', 'https://code.metoffice.gov.uk/svn') }}/ancil/ants/tags/{{ version }}
```